pkg/config/config.go

```diff
@@ -240,7 +242,7 @@ func getenv(name string) string {
 // to the flagset before parsing them with the values specified by
 // args.
 func Load(fs *flag.FlagSet, args []string) (*Config, error) {
-	return load(fs, args, LoadRemote)
+	return load(fs, args, LoadFile)
```

Comment:
I don't see the suggestion button for some reason about what about doing this:

```go
return load(fs, args, func(url string, expand bool, c *Config) error {
  if c.cfg.ExperimentalConfigURLs {
    return LoadRemote(url, expand, c)
  }
  return LoadFile(url, expand, c)
}
```

Reply:
good call, just made this update :+1: